lib/OnboardingMessageProvider.jsm

```diff
@@ -125,20 +173,31 @@ const OnboardingMessageProvider = {
   async translateMessages(messages) {
     let translatedMessages = [];
     for (const msg of messages) {
-      let translatedMessage = msg;
       // If the message has no content, do not attempt to translate it
       if (!msg.content) {
         translatedMessages.push(msg);
         continue;
       }
-      const [button_string, title_string, text_string] = await L10N.formatMessages([
-        {id: msg.content.button_label.string_id},
+
+      let translatedMessage = msg;
```

Comment:
Any chance you'd be modifying a source object here? do you need to make a copy with `{...msg}`?

Reply:
I'll change it just in case.... you can never be too sure with javascript